handshake.go

```diff
@@ -103,6 +103,7 @@ func (srv *Server) writeParameters(ctx context.Context, writer *buffer.Writer, p
 
 	params[ParamServerEncoding] = "UTF8"
 	params[ParamClientEncoding] = "UTF8"
+	params[ParamServerVersion] = srv.Version
```

Comment:
I would like to suggest to only set the version once the version is set.

Reply:
@jeroenrinzema updated